packages/upload/src/vaadin-upload-mixin.js

```diff
@@ -183,7 +185,7 @@ export const UploadMixin = (superClass) =>
           notify: true,
           readOnly: true,
           reflectToAttribute: true,
-          computed: '_maxFilesAdded(maxFiles, files.length)',
```

Comment:
This wasn't working properly so I changed to use the multi-property observer instead.